include/secp256k1.h

```diff
@@ -194,13 +192,17 @@ typedef int (*secp256k1_nonce_function)(
 #define SECP256K1_FLAGS_BIT_CONTEXT_DECLASSIFY (1 << 10)
 #define SECP256K1_FLAGS_BIT_COMPRESSION (1 << 8)
 
-/** Flags to pass to secp256k1_context_create, secp256k1_context_preallocated_size, and
+/** Context flags to pass to secp256k1_context_create, secp256k1_context_preallocated_size, and
  *  secp256k1_context_preallocated_create. */
-#define SECP256K1_CONTEXT_VERIFY (SECP256K1_FLAGS_TYPE_CONTEXT | SECP256K1_FLAGS_BIT_CONTEXT_VERIFY)
 #define SECP256K1_CONTEXT_SIGN (SECP256K1_FLAGS_TYPE_CONTEXT | SECP256K1_FLAGS_BIT_CONTEXT_SIGN)
-#define SECP256K1_CONTEXT_DECLASSIFY (SECP256K1_FLAGS_TYPE_CONTEXT | SECP256K1_FLAGS_BIT_CONTEXT_DECLASSIFY)
+
+/** Deprecated context flags. These flags are treated equivalent to SECP256K1_CONTEXT_SIGN. */
```

Comment:
Unfortunately, it's not possible to apply a `deprecated` attribute to a macro. We could turn this into a `static const` but this potentially breaks existing code. Maybe something to consider for the next API version.

Reply:
It would be easier for other language bindings if these were non-`static` `const`s ... as either macros or statics they aren't exported so we just have to reproduce the values in the target language.